### DISCUSSION TITLE
2018/02/03/video-showing-the-remains-of-beacon-32-needle-peak-nv

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#3] [Announcements]
# 2018/02/03/video-showing-the-remains-of-beacon-32-needle-peak-nv

http://localhost:4000/2018/02/03/video-showing-the-remains-of-beacon-32-needle-peak-nv

**Comment by jengalas** [comment]
Posted by Monte Hendricks
February 19, 2021 at 9:52 pm

_[Imported from WordPress]_

I was pleased to find our video posted here. My wife Julie & I look for excuses to explore remote areas and over the last few years have located and visited the beacon sites in the remote stretch between Fernley and Battle Mountain Nevada. We have photographed the sites and would appreciate sharing information. I’m most curious on who were the people maintaining the beacons in this remote area.

**Comment by jengalas** [comment]
Posted by Dave DuPree
February 6, 2018 at 2:38 pm

_[Imported from WordPress]_

Beautiful scenery and a very interesting airway site. There appear to be remnants of two different beacon towers – presumably an early tower (“322 Mile Blinker”?) and its replacement. The first tower may have been an early windmill-type tower known to have been used by the Post Office Department. That unusual ladder section is a clue. Most of the early POD Air Mail beacon towers were supplied by the Baker Manufacturing Co. They are readily identifiable and photographs are plentiful. However, there was another windmill-type tower used on the airways – though so far, I have found only one photo. I haven’t yet been able to identify the manufacturer. The photo was taken and published in 1932 on “Red Butte mountain” just east of Salt Lake City near the mouth of Emigration Canyon. The occasion of the photo was the replacement of the old Air Mail tower and early 24″ beacon (circa 1925) with a new Dept. of Commerce IDECO tower and more modern 36″ beacon. Something similar may have happened on Needle Peak, although in this case, the old tower was apparently supporting a blinker. That ladder section is definitely not a match for any of the Baker or IDECO towers. However, it could be a match for the other, unidentified POD tower. With only one poor-quality photo, I certainly can’t be sure, but its worth further investigation.

My thanks to Jack for posting the link, and to “ski3pin” for making the trek and posting the video. And in particular, thank you Zhanna, for hosting.
